src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -544,7 +742,7 @@ def testFileDeletion(self):
             Intended to cover the batch deletion of items in the AWSJobStore, but it doesn't hurt
             running it on the other job stores.
             """
-            master = self.master
+            master = self.jobstore_initialized
```

Comment:
It seems like this 'master' variable serves no purpose. It should probably be deleted and it's use replaced with 'self.jobstore_initialized'